libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -211,6 +212,10 @@ class AP_Airspeed
     // get aggregate calibration state for the Airspeed library:
     CalibrationState get_calibration_state() const;
 
+#if HAL_GCS_ENABLED
+    void send_mavlink_airspeed(mavlink_channel_t chan);
```

Comment:
Please pass the link& through rather than the channel number - it means we don't need the GCS.h header